standalone-metastore/src/main/thrift/hive_metastore.thrift

```diff
@@ -893,9 +884,11 @@ struct GetValidWriteIdsResponse {
 
 // Request msg to allocate table write ids for the given list of txns
 struct AllocateTableWriteIdsRequest {
-    1: required list<i64> txnIds,
+    1: optional list<i64> txnIds,
```

Comment:
Optional members can be moved to bottom and keep repl related members together.

Reply:
txnIds is not replication related ..it will be used by normal flow ..so have kept it at the starting

Reply:
done